docs/src/content/docs/for-administrators/my-first-loculus.md

```diff
@@ -134,50 +126,48 @@ Loculus ships with some default organisms, but you probably want to overwrite th
 Let's edit the `custom_values.yaml` file to the following:
 
 ```yaml
```

Comment:
IMO it probably makes sense to keep this consistent with our yaml files which seem to have tabs as two spaces

Reply:
I made it 2 spaces to save some horizontal space as the github actions yamls can get quite nested - maybe prettier doesn't support that type of configuration in md.

Reply:
ok, I added an ignore to avoid formatting this.